### PR TITLE
Populating site with content

### PR DESCRIPTION
**Pull request by prtmsh**
### Changes made:

- Logo image added
- Added Descriptions to Product page/frame2
- Added content to Home page/frame3
- New text for Home page/frame1 in container

### CODE REVIEW

File: src/components/Layout/Header.tsx
Comment (cjds) [inline src/components/Layout/Header.tsx:1]: Can we keep folder structure all lower case?

File: src/components/Layout/Header.tsx [hunk 1]
```diff
@@ -17,10 +18,9 @@ const HeaderContainer = styled(motion.header)`
   z-index: 1000;
 `;
 
-const Logo = styled(motion.h1)`
-  font-size: 2rem;
-  color: ${props => props.theme.colors.primary};
-  font-family: ${props => props.theme.fonts.headings};
+const Logo = styled(motion.img)`
+  width: 100px;
+  height: auto;
```
Comment (cjds) [inline src/components/Layout/Header.tsx:24]: Hmm.. why pixels? Why not a responsive unit ?

File: src/components/Mainscreen/Frame1.tsx [hunk 2]
```diff
@@ -23,25 +23,36 @@ const GradientOverlay = styled(motion.div)`
...
 const ContentWrapper = styled(motion.div)`
-  text-align: center;
   z-index: 2;
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
 `;
 
 const AnimatedText = styled(motion.h1)`
-  font-size: 6rem;
```
Comment (cjds) [inline src/components/Mainscreen/Frame1.tsx:32]: Why this comment?

**Comment by prtmsh** [comment]
Made the required changes!

File: src/components/Layout/Header.tsx
Comment (prtmsh) [reply src/components/Layout/Header.tsx:1]: Just the folders, right? Or files too?

**Review by cjds — APPROVED** [review]

**Pull request merged**